content/en-US/api/CanvasRenderingContext2D.reset.md

```diff
@@ -0,0 +1,111 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.reset'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/reset
+specifications: [[https://html.spec.whatwg.org/multipage/canvas.html]]#[[dom-context-2d-reset]]
+browser_compatibility: api.CanvasRenderingContext2D.reset
+---
+
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/reset](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/reset).**
+
+## Description
+
+The `reset()` method of the `CanvasRenderingContext2D` interface
+resets the rendering context to its default state. This includes setting the backing buffer to transparent black, clearing the saved state stack, clearing the default path and resetting the drawing state to its initial values.
+
+Drawing state consists of:
+ - The current transformation matrix.
+ - The current clipping region.
+ - The current values of the following attributes: 
+    - `strokeStyle`
+    - `fillStyle`
+    - `globalAlpha`
+    - `lineWidth`
+    - `lineCap`
+    - `lineJoin`
+    - `miterLimit`
+    - `lineDashOffset`
+    - `shadowOffsetX`
+    - `shadowOffsetY`
+    - `shadowBlur`
+    - `shadowColor`
+    - `filter`
+    - `globalCompositeOperation`
+    - `font`
+    - `textAlign`
+    - `textBaseline`
+    - `direction`
+    - `textLetterSpacing`
+    - `textWordSpacing`
+    - `fontKerning`
+    - `fontStretch`
+    - `fontVariantCaps`
+    - `textRendering`
+    - `imageSmoothingEnabled`
+    - `imageSmoothingQuality`
+ - The current dash list.
```

Comment:
I'm glad to have this captured. Down the line we'll move this information into what will be a higher level page on MDN.

Reply:
👍